controllers/storagecluster/reconcile.go

```diff
@@ -532,6 +532,12 @@ func (r *StorageClusterReconciler) reconcilePhases(
 		}
 	}
 
+	// For security of encrypted messages, When both encryption and compression are enabled, compression setting will be ignored and message will not be compressed.
+	networkSpec := instance.Spec.Network
+	if networkSpec != nil && networkSpec.Connections != nil && networkSpec.Connections.Encryption != nil && networkSpec.Connections.Encryption.Enabled && networkSpec.Connections.Compression != nil && networkSpec.Connections.Compression.Enabled {
```

Comment:
```suggestion
	if networkSpec != nil && networkSpec.Connections != nil && 
	    networkSpec.Connections.Encryption != nil && networkSpec.Connections.Encryption.Enabled && 
	    networkSpec.Connections.Compression != nil && networkSpec.Connections.Compression.Enabled {
```
Can we break this into multiple lines something like this?